app/src/main/java/org/simple/clinic/newentry/PatientEntryModule.kt

```diff
@@ -0,0 +1,21 @@
+package org.simple.clinic.newentry
+
+import dagger.Module
+import dagger.Provides
+import org.simple.clinic.appconfig.Country
+import org.simple.clinic.newentry.country.InputFieldsFactory
+import org.simple.clinic.newentry.form.InputField
+import org.threeten.bp.LocalDate
+import org.threeten.bp.format.DateTimeFormatter
+import javax.inject.Named
+
+@Module
+class PatientEntryModule {
+  @Provides
+  fun inputFieldsFactory(@Named("date_for_user_input") dateTimeFormatter: DateTimeFormatter): InputFieldsFactory =
```

Comment:
Can we pass in the `userClock` as well and change the `LocalDate.now()` to take it in as a parameter?

Reply:
Done.